bucket/dngrep.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "2.9.482.0",
+    "description": "Graphical GREP tool to search across text files, Word and Excel documents, PDFs and archives",
+    "homepage": "http://dngrep.github.io/",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dnGrep/dnGrep/releases/download/v2.9.482.0/dnGREP.2.9.482.x64.msi",
+            "hash": "59cadfa678eca183ebdd9840455da3b3ac9872eb69b39d8175bc6594ebe0532f"
+        },
+        "32bit": {
+            "url": "https://github.com/dnGrep/dnGrep/releases/download/v2.9.482.0/dnGREP.2.9.482.x86.msi",
+            "hash": "ffe7136771bd90fb884587b89341a64a2fd2b0cc4008e55222cd496e47a9d813"
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\dnGREP.Settings.dat\")) { New-Item \"$dir\\dnGREP.Settings.dat\" | Out-Null }",
```

Comment:
Aren't .dat files binary formatted? Can you confirm if dnGrep is able to read/write to this file when it's created by Scoop here?

Reply:
it's actually a xml file.
I can confirm that after creating an empty file with this name and moving to persist folder, when opening Options windows in dnGREP gui, it populates this file with default values.

Reply:
Okay, thanks for the info!